intro-HPC/ch_account.tex

```diff
@@ -386,6 +378,10 @@ \section{First Time connection to the \hpc}
     \item  The first time you make a connection to the loginnode, a Security
       Alert will appear and you will be asked to verify the authenticity of the
       loginnode.
+      \ifgent
+      Make sure the fingerprint in the Alert matches \strong{2f:0c:f7:76:87:57:f7:5d:2d:7b:d1:a1:e1:86:19:f3}
+      \fi
+      %TODO: fingerprints for other institution login nodes?
```

Comment:
there's a macro for the fingerprint in #89, so ok to assume this is going to be fixed there?